### PR TITLE
Testing for Evented module

### PR DESCRIPTION
**Pull request by chuckcarpenter**
this branch adds some tests for the base class and increases code coverage

```
TOTAL: 21 SUCCESS
-------------|----------|----------|----------|----------|-------------------|
File         |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-------------|----------|----------|----------|----------|-------------------|
All files    |    72.01 |    59.19 |       64 |    72.78 |                   |
 evented.js  |    84.85 |    80.77 |      100 |    84.85 |    30,31,32,33,35 |
 shepherd.js |      100 |      100 |      100 |      100 |                   |
 step.js     |    66.67 |    55.12 |       40 |    68.02 |... 84,385,400,401 |
 tour.js     |    73.81 |       50 |      100 |    73.49 |... 61,165,166,167 |
 utils.js    |    90.48 |     87.5 |      100 |    90.48 |             34,52 |
-------------|----------|----------|----------|----------|-------------------|
```

### CODE REVIEW

File: src/js/evented.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class Evented {
 
   off(event, handler) {
     if (typeof this.bindings === 'undefined' || typeof this.bindings[event] === 'undefined') {
-      return;
+      return false;
```
Comment (rwwagner90) [inline src/js/evented.js]: Why did we change to return false here? Not saying it's wrong, just want to understand what it does for us.

Comment (chuckcarpenter) [reply src/js/evented.js]: clearer response for tests

Comment (rwwagner90) [reply src/js/evented.js]: I wonder if maybe we should skip the return part, and just wrap the whole method in 
```js
if (!_.isUndefined(this.bindings) && !_.isUndefined(this.bindings[event])) {
```

Comment (rwwagner90) [reply src/js/evented.js]: Perhaps we should save that for a later refactor though, since we might replace evented anyway

**Pull request merged**